src/dispatchers/dispatcher.ts

```diff
@@ -121,15 +122,21 @@ export class Dispatcher<Type extends { guid: string }, Initializer> extends Even
 }
 
 export type DispatcherScope = Dispatcher<any, any>;
-class Root extends Dispatcher<{ guid: '' }, {}> {
-  constructor(connection: DispatcherConnection) {
+export class Root extends Dispatcher<{ guid: '' }, {}> {
+  constructor(connection: DispatcherConnection, private readonly createPlaywright?: (scope: DispatcherScope) => Promise<PlaywrightDispatcher>) {
     super(connection, { guid: '' }, '', {}, true);
   }
+
+  async initialize(params: { language?: string }): Promise<channels.RootInitializeResult> {
+    assert(this.createPlaywright);
+    return {
+      playwright: await this.createPlaywright(this),
```

Comment:
- Let's throw if this method is called twice.
- Why pass `scope` if everyone already has access to the `Root` anyway?